avocado/utils/configure_network.py

```diff
@@ -189,6 +189,29 @@ def get_peer_interface(self, peer_ip):
         else:
             return peer_interface
 
+    def ping6_check(interface, count, option=None, flood=False):
+        """
+        IPV6 ping check to peer
+        """
+        cmd = "ip addr show %s" % interface
+        try:
+            for line in self.session.cmd(cmd).stdout.decode("utf-8") \
+                                                    .splitlines():
+                if "inet6" in line:
+                    ipv6 = line.split()[1].split('/')[0]
+        except Exception as ex:
```

Comment:
Suggestion: This exception is too generic. Can we try to make it more specific? (or with multiple except statements)